Magento_Theme/layout/default.xml

```diff
@@ -17,8 +17,8 @@
         <referenceBlock name="logo">
             <arguments>
                 <argument name="logo_file" xsi:type="string">images/logo.png</argument>
-                <argument name="logo_img_width" xsi:type="number">220</argument>
-                <argument name="logo_img_height" xsi:type="number">60</argument>
+                <argument name="logo_img_width" xsi:type="number">272</argument>
```

Comment:
Not really relevant for this PR, but I wonder if it's possible in Magento -- on the research frontend I put in a breakpoint rule that sets the logo to `width: 204px` (which is 75%) in mobile view.  This makes the header generally take up less vertical space, but also fixed a problem with the hamburger popping up above the logo instead of to the right of the logo.